CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/).
 ### Fixed
 
 - fix tests which relied on a third party reporter in jcenter [#772](https://github.com/JLLeitschuh/ktlint-gradle/pull/772)
+- Remove IntelliJ plugin references from README [#775](https://github.com/JLLeitschuh/ktlint-gradle/pull/775)
```

Comment:
This line should be under the [Unreleased] Header
Thanks. sorry for the delay in review